.tav.yml

```diff
@@ -157,6 +157,27 @@ mongodb:
   node: '>=14.20.1'
   commands: node test/instrumentation/modules/mongodb.test.js
 
+# mongoose
+# - v5 is still maintained (v5.13.14 on 2021-12-27). v5.7.0 switched from `mongodb-core`
+#   to `mongodb`. Testing we are not affected by the switch with a subset of 294 releases
+# - v6 has 94 releases. Testing a subset
+# - v7 was released recently (2023-02-27)
+mongoose-5:
+  name: mongoose
+  versions: '5.0.0 || 5.3.2 || 5.5.10 || 5.9.2 || 5.11.2 || 5.13.14 || 5.13.17'
+  node: '>=4.0.0'
+  commands: node test/instrumentation/modules/mongoose.test.js
+mongoose-6:
+  name: mongoose
+  versions: '6.0.0 || 6.1.2 || 6.2.9 || 6.4.5 || 6.7.1 || 6.10.4 || 6.11.1'
+  node: '>=12.0.0'
+  commands: node test/instrumentation/modules/mongoose.test.js
+mongoose-7:
+  name: mongoose
+  versions: '7.0.0 || 7.0.4 || 7.1.0 || 7.1.2 || 7.2.1 || 7.2.2'
```

Comment:
Probably want a `... <8` specifier on here so we test new 7.x releases.  Probably for the 5.x and 6.x series as well.